sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxSqaleXmlLoader.java

```diff
@@ -0,0 +1,186 @@
+/*

```

Comment:
i would prefer not to include this. its likely better to save the xml definition into temp file and use SQ class. having this is we might end up braking our sqale model eveytime core changes something.